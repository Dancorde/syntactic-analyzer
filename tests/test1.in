program lalg;
{entrada}
var a: integer;
begin
readd(a, 1.4, 1);
end.