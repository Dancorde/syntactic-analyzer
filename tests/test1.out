program - program
lalg - id
; - ;
var - var
a - id
: - :
integer - integer
; - ;
begin - begin
readd - id
( - (
a - id
, - ,
1.4 - numero_real
, - ,
1 - numero_int
) - )
; - ;
end - end
. - .
